eng/testing/performance/performance-setup.ps1

```diff
@@ -152,7 +152,12 @@ if ($iOSMono) {
     {
         mkdir $WorkItemDirectory
     }
-    Copy-Item -path "$SourceDirectory\iosHelloWorld\nollvm" $PayloadDirectory\iosHelloWorld\nollvm -Recurse
+    if($iOSLlvmBuild) {
```

Comment:
Do we mean to copy  "$SourceDirectory\iosHelloWorld\nollvm" to $PayloadDirectory\iosHelloWorld\nollvm and "$SourceDirectory\iosHelloWorld\llvm" to $PayloadDirectory\iosHelloWorld\llvm  instead?